app/templates/400.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load static %}
```

Comment:
If I check correctly, you don't need static here.

Reply:
Okay thanks, will remove